es-models/gdc_from_graph/case.mapping.yaml

```diff
@@ -48,10 +48,18 @@ properties:
     type: keyword
   days_to_index:
     type: long
+  days_to_lost_to_followup:
+    type: long
   demographic:
     properties:
+      cause_of_death:
+        type: keyword
       created_datetime:
         type: keyword
+      days_to_birth:
+        type: long
+      days_to_death:
```

Comment:
This field (and some others as well) previously exists under `diagnoses`, it seems like it still exists there, is it intended to duplicate it? Or the old field will be removed eventually?